browser_patches/firefox-beta/juggler/protocol/PageHandler.js

```diff
@@ -302,8 +302,59 @@ class PageHandler {
     return await this._contentPage.send('adoptNode', options);
   }
 
-  async ['Page.screenshot'](options) {
-    return await this._contentPage.send('screenshot', options);
+  async ['Page.screenshot']({ mimeType, clip, omitDeviceScaleFactor }) {
+    let lastError = null;
+    for (let i = 0; i < 3; ++i) {
+      try {
+        // This might occasionally throw if called during navigation.
```

Comment:
I think we should solve this on the client side, because `3` is such a random number. Perhaps 10000 would be ok?